src/amo/components/SearchTools/index.js

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import Search from 'amo/components/Search';
+import {
+  ADDON_TYPE_OPENSEARCH,
+  SEARCH_SORT_RELEVANCE,
+} from 'core/constants';
+import { convertFiltersToQueryParams } from 'core/searchUtils';
+
+type Props = {|
+  filters: Object,
```

Comment:
Arguably this should use `import type { SearchFilters} from 'amo/components/AutoSearchInput';`... which I know is just an object but would make it easier to replace with a better type later.

That said I think we still have to hunt for other instances of filter without a type and that's probably not even the right place for that type, so 🤷‍♂️